app/src/org/commcare/preferences/CommCarePreferences.java

```diff
@@ -227,6 +225,45 @@ public boolean onPreferenceClick(Preference preference) {
         } else {
             getPreferenceScreen().removePreference(analyticsButton);
         }
+
+        Preference printTemplateSetting = findPreference(PREFS_PRINT_DOC_LOCATION);
```

Comment:
Yea it is - once I needed to be calling `setupButtons()` from within `setVisibilityOfUpdateOptionsPref()`, it made me realize that it makes more sense to have the functionality for setting the onClick method for the print template be in that method as well, instead of having it be randomly separate.